Vokoder.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "Vokoder"
-  s.version          = "4.0.2"
+  s.version          = "4.1.0"
```

Comment:
My thought was that while there are some Swift renaming changes, there aren't any breaking changes in terms of functionality, or the underlying obj-c code.